### DISCUSSION TITLE
How to create an issue against a code snippet

### DISCUSSION THREAD

**Discussion by mahedee** [OpenCodeFoundation/eSchool#369] [Q&A] [answered]
I have found some code snippet or code block or statement which I don't understand or I think useless. How can I create an issue/discussion against that code block. Please give me with an example. Reference link also appreciated.

**Comment by ratanparai** [comment, marked as answer]
@mahedee: You can use code snippet in your issue or discussion. for example -

https://github.com/OpenCodeFoundation/eSchool/blob/8f9a75effe3b27d8925ab426451f45113dcc4d44/src/Services/Enrolling/Enrolling.API/graphql/EnrollingQuery.cs#L14-L18

Please check https://docs.github.com/en/github/managing-your-work-on-github/creating-a-permanent-link-to-a-code-snippet for guideline.
